### PR TITLE
Upgrade setup

### PR DESCRIPTION
**Pull request by LilSpazJoekp**

### CODE REVIEW

File: tests/integration/__init__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import pytest
 from betamax.cassette import Cassette
 
-from .utils import (
+from ..utils import (
```
Comment (bboe) [inline tests/integration/__init__.py:9]: I don't think that this change is correct as there is a separate `utils.py` in `tests/integration/`

Comment (bboe) [reply tests/integration/__init__.py:9]: Oh nevermind. I see that you relocated that file. Are we using that util for anything other than integration tests?

**Review by bboe — APPROVED** [review]
One question about the utils move. It's not a deal breaker though so I'm approving.

File: tests/integration/__init__.py (hunk 1 above)
Comment (LilSpazJoekp) [reply tests/integration/__init__.py:9]: We're not but the ruff checks gets mad about having a module not prefixed with `test_` in the test folders.

Comment (bboe) [reply tests/integration/__init__.py:9]: Okay, that makes sense then.

**Pull request merged**